.gitignore

```diff
@@ -4,3 +4,6 @@ zig-cache/
 *.pdf
 *.dtb
 rust/target/
+*overlayed.cpio.gz
```

Comment:
Do we need this in the gitignore? If it's a build artefact I would think it would be in the build directory right?

Reply:
For some reason it gets put in `board/` in both `examples/virtio` and `examples/virtio-snd`

Reply:
Fixed in https://github.com/au-ts/libvmm/commit/ea9d281a633201b0634762deb141e46379032724